test/test_files/extension/extension.test

```diff
@@ -25,3 +25,15 @@ Waterloo|150000
 -STATEMENT INSTALL http;
 ---- error
 IO exception: HTTP Returns: 404, Failed to download extension: "http" from extension.kuzudb.com/v0.1.0/osx_arm64/libhttp.kuzu_extension.
+
+-CASE InstallUnofficialExtensions
```

Comment:
Seems no need to start a new case right?

Reply:
Each test case is running in parallel. Making thoses tests separate tests allows us to gain more parallelism while testing.